src/plugins/irc-events/connection.js

```diff
@@ -202,15 +202,12 @@ module.exports = function (irc, network) {
 		data.options.PREFIX.forEach((mode) => {
 			network.prefixLookup[mode.mode] = mode.symbol;
```

Comment:
This can probably be removed and use `serverOptions.PREFIX` instead.

Reply:
tried that, the problem is that it makes the compare function awkward. All callsites only have the prefix string (@, + etc) and then you'd need to shoehorn those strings into the expected object which looked very bad when I tried it

Reply:
wait, wrong part of the code.

Let me check where we even use the prefixLookup